<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWindow</class>
 <widget class="QWidget" name="RegistrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>365</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>365</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>365</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Open Sans Condensed SemiBold</family>
    <pointsize>14</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>RegistrationWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(233, 234, 255)</string>
  </property>
  <widget class="QLabel" name="newLoginLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Condensed SemiBold</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Логин</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="newLoginLineEdit">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>70</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Condensed SemiBold</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: none;
    border-radius: 5px;
}

QLineEdit:focus {
    border: 2px solid blue;
}
</string>
   </property>
   <property name="placeholderText">
    <string>Введите логин</string>
   </property>
  </widget>
  <widget class="QLabel" name="newPasswordLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Condensed SemiBold</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>160</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Condensed SemiBold</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: none;
    border-radius: 5px;
}

QLineEdit:focus {
    border: 2px solid blue;
}</string>
   </property>
   <property name="placeholderText">
    <string>Введите пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPasswordLineEdit2">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>210</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Condensed SemiBold</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: none;
    border-radius: 5px;
}

QLineEdit:focus {
    border: 2px solid blue;
}</string>
   </property>
   <property name="placeholderText">
    <string>Повторите пароль</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="newPasswordCheckBox2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Condensed SemiBold</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="newPasswordCheckBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Condensed SemiBold</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="newRegistrationButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>176</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Condensed SemiBold</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border-radius: 5px;
}

QPushButton:hover {
    border: 2px solid blue;
}
</string>
   </property>
   <property name="text">
    <string>Зарегистрироваться</string>
   </property>
  </widget>
  <widget class="QLabel" name="newRegInfoLabel">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>305</y>
     <width>350</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Condensed SemiBold</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
